test/e2e/e2e_test.go

```diff
@@ -72,7 +72,7 @@ func TestMasterMachineCreated(t *testing.T) {
 func createTestClients() (*Clients, error) {
 	kubeconfig := os.Getenv("KUBE_CONFIG")
 	if kubeconfig == "" {
-		return nil, fmt.Errorf("KUBE_CONFIG environment variable is not set")
+		t.Skip("KUBE_CONFIG environment variable is not set")
```

Comment:
Where is `t` defined? I think you need to pass that to the `createTestClients()` method.

Reply:
I have a preference for performing this  if check in L44. If one is able to retrieve the `kubeconfig` then we can pass that as a parameter to `createTestClients()`